src/components/users/edit-user/edit-user.component.js

```diff
@@ -1,13 +1,56 @@
 import React, { Component } from "react";
 
+import { withRouter } from "react-router";
+import LoadingPanel from "./../../partials/loading-panel/loading-panel.component";
+
+import { bufferToBase64 } from "./../../../helpers/get-data";
+
+import UserService from "./../../../services/user.service";
+import StitchService from "./../../../services/stitch.service";
+
+const userService = UserService.getInstance();
+const stitchService = StitchService.getInstance();
+
 class EditUser extends Component {
+  state = {src: "", isLoaded: false};
+
+
+  componentDidMount(){
+    const id = this.props.match.params.id;
+    userService.getUserById(id).then((user)=>{
+      if (user.profilePic){
+        stitchService.getPhoto(user.profilePic).then((pic)=>{
+          if (pic.pictureURL){
+            this.setState({isLoaded: true, src: pic.pictureURL});
+          } else {
+            if (pic && (pic.picture || pic.photo || pic.thumbNail)){
+              pic = pic.thumbNail ? pic.thumbNail : (pic.picture ? pic.picture : pic.photo);
+              this.setState({isLoaded: true, src: "data:image/jpeg;base64," + bufferToBase64(pic.buffer)});
+            } else {
+              this.setState({isLoaded: true});
+            }
```

Comment:
I've cleaned up the `Photo` collection so that there are no longer any documents containing `photo` and so it's safe to remove that from the checks.